spec/tests/angularComponent/ngcOmniboxControllerSpec.js

```diff
@@ -8,6 +8,7 @@ describe('ngcOmnibox.angularComponent.ngcOmniboxController', () => {
 
     omniboxController = new NgcOmniboxController();
     omniboxController._suggestionElements = [fakeEl, fakeEl, fakeEl, fakeEl];
+    omniboxController.isSelectable = () => true;
```

Comment:
I just need it to be a defined function. It could be a `noop` but returning true seemed fine too.